bitnami/sealed-secrets/templates/rolebinding.yaml

```diff
@@ -64,11 +64,11 @@ metadata:
 roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: Role
-  name: {{ printf "%s" (include "sealed-secrets.namespacedRoleName" .) }}
+  name: {{ printf "%s" (include "sealed-secrets.namespacedRoleName" $) }}
 subjects:
   - apiGroup: ""
     kind: ServiceAccount
     name: {{ include "sealed-secrets.serviceAccountName" $ }}
-    namespace: {{ include "sealed-secrets.namespace" $ }}
+    namespace: {{ include "common.names.namespace" $ | quote }}
```

Comment:
Then we can remove `sealed-secrets.namespace` function from the `_helpers.tpl`, right?

Reply:
I can still find 23 matches for the include "sealed-secrets.namespace" in other files.
Maybe it's better to replace those usages including this one in a separate PR.

What do you think?